src/io/flutter/inspections/SdkConfigurationNotificationProvider.java

```diff
@@ -108,6 +117,21 @@ public EditorNotificationPanel createNotificationPanel(@NotNull VirtualFile file
     return null;
   }
 
+  private EditorNotificationPanel createOutOfDateFlutterSdkPanel() {
+
+    final FlutterSettings settings = FlutterSettings.getInstance(project);
+    if (settings == null || settings.ignoreOutOfDateSdks()) return null;
+
+    EditorNotificationPanel panel = new EditorNotificationPanel();
+    panel.setText(FlutterBundle.message("flutter.old.sdk.warning"));
+    panel.createActionLabel("Dismiss", () -> {
+      settings.setIgnoreOutOfDateSdks(true);
```

Comment:
It doesn't yet.  My thinking is that we'll want to introduce some kind of event that blanks the setting.  Not sure what that should be just yet though.  Maybe the next time the min version bumps?

I admit to have thought about it just not settled yet so I punted.  Thoughts welcome!